rclpy/CMakeLists.txt

```diff
@@ -176,6 +176,7 @@ if(BUILD_TESTING)
       test/test_node.py
       test/test_parameter.py
       test/test_publisher.py
+      test/test_parameter_service.py
```

Comment:
alphabet order

Reply:
this needs to be addressed? @Barry-Xu-2018

Reply:
Addressed it in https://github.com/ros2/rclpy/pull/1107

Reply:
There is a problem. 
Repeated test/test_parameter_service.py in CMakefile.  But https://github.com/ros2/rclpy/pull/1107/commits/3c3c36cc692bfb9d891f6042640697f8121581be didn't do this.  
Anyway, I will make new PR to remove it.